runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -186,6 +190,7 @@ void addKill(final LootTrackerRecord record)
 		}
 
 		kills += record.getKills();
+		records.add(record);
```

Comment:
Is it possible to do this in a way which doesn't require storing all of the records on each box? Usually they get combined with exiting records which uses less memory, which I like.